app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -77,6 +84,12 @@ protected void onCreate(Bundle savedInstanceState) {
         final ImageButton left = findViewById(R.id.leftBtn);
         final ImageButton right = findViewById(R.id.rightBtn);
 
+        // invisible four buttons in case joysick method doesn't work
+        forward.setVisibility(View.GONE);
```

Comment:
Could you comment out the buttons directly in the xml file instead and explain there, seeing as there isn't code for them currently in java?

Reply:
Also, while doing this, comment out the button listeners for the left, right, forward, back in MainActivity, as they aren't being used.